File: integration-tests/maven/src/test/resources-filtered/projects/rr-external-artifacts/external-lib/src/main/java/org/acme/lib/GreetingResource.java
package org.acme.lib;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;

@Path("/greet")
class GreetingResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String greet() {
        return "Hello from Quarkus REST";
    }
}

File: integration-tests/gradle/src/main/resources/grpc-multi-module-no-java/application/src/main/kotlin/org/acme/GreetingResource.kt
package org.acme

import io.quarkus.grpc.GrpcClient
import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType
import org.acme.module1.SomeClass1
import org.acme.proto.Greeter

@Path("/version")
class GreetingResource {

    @GrpcClient
    lateinit var hello: Greeter

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun getVersion(): String {
        return SomeClass1().getVersion()
    }
}

